pkgs/desktops/gnome-3/3.16/core/gnome-control-center/default.nix

```diff
@@ -17,7 +18,8 @@ stdenv.mkDerivation rec {
     sha256 = "07vvmnqjjcc0cblpr6cdmg3693hihpjrq3q30mm3q68pdyfzbjgf";
   };
 
-  propagatedUserEnvPkgs = [ gnome3.gnome_themes_standard gnome3.libgnomekbd ];
+  propagatedUserEnvPkgs =
+    [ gnome3.gnome_themes_standard gnome3.libgnomekbd sound-theme-freedesktop ];
```

Comment:
Can you explain me what's the problem in not changing this line? The change below to XDG_DATA_DIRS should be enough.